code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -2,25 +2,12 @@
 	var/obj/item/bodypart/BP = get_bodypart(check_zone(user.zone_selected))
 	if(!BP)
 		return ..()
-	var/painless = (HAS_TRAIT(user, TRAIT_ANALGESIA) || HAS_TRAIT(user, TRAIT_PAIN_RESIST))
+	// var/painless = (HAS_TRAIT(user, TRAIT_ANALGESIA) || HAS_TRAIT(user, TRAIT_PAIN_RESIST))
 	if(W.tool_behaviour == TOOL_WELDER && IS_ROBOTIC_LIMB(BP) && BP.brute_dam) //prioritize healing if we're synthetic
 		return ..()
-	if(user.a_intent != INTENT_HELP || !W.get_temperature() || !BP.can_bandage()) //this will also catch low damage synthetic welding
+	if(user.a_intent != INTENT_HELP || !W.get_temperature()) //this will also catch low damage synthetic welding
 		return ..()
 	. = TRUE
```

Comment:
This proc was just for the old cauterizing system and currently blocks the new one from functioning